glfw.sh

```diff
@@ -0,0 +1,31 @@
+package: GLFW
+version: "3.3"
+source: https://github.com/glfw/glfw.git
```

Comment:
I guess you need at least a `build_requires` on `CMake` and `GCC-ToolChain`?

Reply:
yep. 

Did not notice the GCC-ToolChain is missing as my zshrc on my CentOS dev. machine has : 

[[ -a /opt/rh/devtoolset-7/enable ]] && source /opt/rh/devtoolset-7/enable

in it...